exercises/practice/high-scores/.meta/example.cpp

```diff
@@ -0,0 +1,32 @@
+#include <algorithm>
+
+#include "high_scores.h"
+
+namespace arcade {
+
+vector<int> HighScores::list_scores() {
+    return scores;
+};
+
+int HighScores::latest_score() {
+    return scores.back();
+};
+
+int HighScores::personal_best() {
+    return *max_element(scores.begin(), scores.end());
+};
+
+vector<int> HighScores::top_three() {
+    vector<int> sorted_scores{scores};
+    sort(sorted_scores.begin(), sorted_scores.end(), greater<int>());
+    size_t limit{3};
+    if (scores.size() < limit) limit = scores.size();
+    vector<int> top3{};
```

Comment:
I know this is just an example and a proof that the exercise is solvable but in my experience some students look at this as "the official solution". Do you mind replacing the default-initialization and the loop with an initialization that uses iterators. That would be shorter and IMHO easier to read.

Reply:
Yes, that is probably easier to read. My tired head had a heap in the first iteration and I tried to replace it with something easier. I would be happy to implement a suggestion if you have one ready.

Reply:
How about: `return {begin(sorted_scores), begin(sorted_scores) + limit};`